README.md

```diff
@@ -155,6 +160,12 @@ Mobile Wallet acts as a **self-custodial** Identity Wallet that:
 
 This implementation involves 3 distinct participants that have different responsibilities:
 
+### PFI
+
+The PFI operates a KCC issuer, as well as a tbDEX exchange.
```

Comment:
interesting i've never thought to consider each PFI acting as a _tbdex exchange_. kinda cool.
other thoughts would be: "as a liquidity provider"

Reply:
mmm. I was struggling to describe what it is to run the tbDEX part of PFI. Happy to use "liquidity provider" as it'll align with other stuff